extension/goplugin/schemas_test.go

```diff
@@ -270,13 +271,13 @@ var _ = Describe("Schemas", func() {
 			c, err := testSchema.Count(goext.Filter{}, context)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(c).To(Equal(uint64(0)))
-			for i := 10; i < 12; i++ {
-				createdResource.ID = string(i)
+			for i := 0; i < 2; i++ {
+				createdResource.ID = strconv.Itoa(i)
```

Comment:
`string(0)` becomes weird result. You can try it in golang playground. By using strconv, you do not hit any issue if `i` starts with 0.

Reply:
Fun fact: `string(1)` is also not a useful ID value ;) This used one character strings with the character being `i`th ASCII character. For example `string(65)` results in `"A"`.

Thanks for this change, this was probably the intention of the original author ;)